File: fixtures/scale/dart/small/lib/model/render_options.dart
class RenderOptions {
  final String align;
  final int width;
  final String frameName;

  const RenderOptions({
    this.align = 'left',
    this.width = 60,
    this.frameName = 'ascii',
  });

  RenderOptions copyWith({String? align, int? width, String? frameName}) {
    return RenderOptions(
      align: align ?? this.align,
      width: width ?? this.width,
      frameName: frameName ?? this.frameName,
    );
  }

  int get clampedWidth => width.clamp(8, 200);
}

File: fixtures/scale/java/small/src/textprinter/model/RenderOptions.java
package textprinter.model;

public class RenderOptions {
    private String align = "left";
    private int width = 60;
    private String frameName = "ascii";
    private String themeName = "plain";

    public String getAlign() { return align; }
    public void setAlign(String align) { this.align = align; }

    public int getWidth() { return width; }
    public void setWidth(int width) { this.width = Math.max(8, width); }

    public String getFrameName() { return frameName; }
    public void setFrameName(String frameName) { this.frameName = frameName; }

    public String getThemeName() { return themeName; }
    public void setThemeName(String themeName) { this.themeName = themeName; }
}
